build.gradle

```diff
@@ -65,7 +65,7 @@ artifactory {
             contextUrl = "${baseUrl}/artifactory" as String
         }
         repository {
-            repoKey = System.getenv('ARTIFACTORY_DEPLOYER_REPO_KEY') ?: project.findProperty('NEXUS_REPO_KEY')
+            repoKey = System.getenv('ARTIFACTORY_REPOSITORY_KEY')
```

Comment:
please, specify it in Description of PR (as "Upgrade Notes")

Reply:
Why is the value from the parameters not being used here? 
'''
System.getenv('ARTIFACTORY_REPOSITORY_KEY') ?: project.properties.get('artifactoryRepKey')
'''

Reply:
Agree, `artifactoryRepositoryKey` i think is better